### PR TITLE
Fix integration tests configuration for SWF Diagram Editor

### PR DESCRIPTION
**Pull request by Josephblt**
This change disables the integration Test for macOS builds.

### CODE REVIEW

**Review by paulovmr — APPROVED** [review]

**Review by romartin — APPROVED** [review]

File: packages/serverless-workflow-diagram-editor/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "build:dev:linux:darwin": "mvn clean install -DskipTests -DskipITs=$(build-env global.build.testIT --not) -Pno-showcase && pnpm dist",
     "build:dev:win32": "pnpm powershell \"mvn clean install `-DskipTests `-DskipITs=$(build-env global.build.testIT --not) `-Pno-showcase\" && pnpm dist",
     "build:prod": "run-script-os",
-    "build:prod:linux:darwin": "pnpm lint && mvn clean install -DskipTests=$(build-env global.build.test --not) -DskipITs=$(build-env global.build.testIT --not) -Pno-showcase && pnpm dist",
+    "build:prod:darwin": "pnpm lint && mvn clean install -DskipTests=$(build-env global.build.test --not) -DskipITs=true -Pno-showcase && pnpm dist",
+    "build:prod:linux": "pnpm lint && mvn clean install -DskipTests=$(build-env global.build.test --not) -DskipITs=$(build-env global.build.testIT --not) -Pno-showcase && pnpm dist",
```
Comment (tiagobento) [inline packages/serverless-workflow-diagram-editor/package.json]: Ok, found the issue. The name of the env property changed. Here's the correct changes:

```suggestion
    "build:prod:linux:darwin": "pnpm lint && mvn clean install -DskipTests=$(build-env tests.run --not) -Dmaven.test.failure.ignore=$(build-env tests.ignoreFailures) -DskipITs=$(build-env integrationTests.run --not) -Pno-showcase && pnpm dist",
```

**Review by tiagobento — CHANGES_REQUESTED** [review]

File: packages/serverless-workflow-diagram-editor/package.json (hunk 1 above)
Comment (tiagobento) [reply packages/serverless-workflow-diagram-editor/package.json]: That's a classic example of semantic conflicts. They happen and it's very hard to predict them. I'm glad we found it quickly :) Thanks a lot @Josephblt @romartin

File: packages/serverless-workflow-diagram-editor/package.json [hunk 2]
```diff
@@ -22,7 +22,8 @@
...
     "build:prod:win32": "pnpm lint && pnpm powershell \"mvn clean install `-DskipTests=$(build-env global.build.test --not) `-DskipITs=$(build-env global.build.testIT --not) `-Pno-showcase\" && pnpm dist",
```
Comment (paulovmr) [inline packages/serverless-workflow-diagram-editor/package.json]:
```suggestion
    "build:prod:win32": "pnpm lint && pnpm powershell \"mvn clean install `-DskipTests=$(build-env tests.run --not) `-Dmaven.test.failure.ignore=$(build-env tests.ignoreFailures) `-DskipITs=$(build-env integrationTests.run --not) `-Pno-showcase\" && pnpm dist",
```

**Review by paulovmr — CHANGES_REQUESTED** [review]

**Review by paulovmr — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]
Good to merge when green. Thanks @Josephblt !

**Pull request merged**